src/App.test.jsx

```diff
@@ -1,4 +1,42 @@
+import { render, screen } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom';
+import App from './App';
 
-test('Should render the header', () => {
+const user = {
+  id: 1,
+  created_at: '2021-12-13T00:17:29+00:00',
+  name: 'Vonta',
+  avatar: 'https://thumbs.gfycat.com/NiceRequiredGrunion-size_restricted.gif',
+  header: 'https://static.wikia.nocookie.net/naruto/images/5/50/Team_Kakashi.png',
+  likes: ['React', 'Anime', 'Traveling', 'Living', 'Tower Defense Games', 'Card Games'],
+  motto: 'Res Non Verba',
+  color: 'crimson',
+};
+
+describe('App', () => {
+  it('Should render the header', async () => {
+    render(
+        <MemoryRouter>
+          <App user={user} />
```

Comment:
I don't think you should need the user props for just testing whether the header renders on the page. Also, `MemoryRouter` is only necessary when you are rendering `links` that navigate to other components.

Reply:
That makes sense! I revised my test removing <code>MemoryRouter</code> and user prop. The revised test is pushed to branch "revised".